Add "Set rates" operation (#694)
MewsSystems/gitbook-connector-api: operations/rates.md

| `GroupId` | string | required | Unique identifier of `Rate Group` where the rate belongs. |
| `ServiceId` | string | required | Unique identifier of the `Service`. |
| `BaseRateId` | string | optional | Unique identifier of the base `Rate`. |

> | `IsBaseRate` | boolean | required | Whether the rate is a base rate. |

Reviewer: It's a minor point, and I know we've used "Whether" in the past plenty of times, but I prefer this wording for booleans:
```suggestion
| `IsBaseRate` | boolean | required | Indicates if this is a base rate. |
```

{}
```

> ## Set rates

Reviewer: Will this operation eventually replace `Add rates` or does it serve a different purpose? if we are going to have both, we need to say what the difference is, i.e. when would you use this operation compared to the other operation?

Author: This one has "upsert" behaviour - it can add new entities or update the existing ones. "Add" operation is just for creation of new resources. We usually have "add" and "update", but this one was geared towards Choice's requirements specifically.

| `Type` | [Rate Add Type](rates.md#rate-add-type) | required | Type of the rate. |
| `AccountingCategoryId` | string | optional | Unique identifier of the accounting category the rate belongs to. |
| `BusinessSegmentId` | string | optional | Unique identifier of the business segment. |
| `Names` | [Localized text](_objects.md#localized-text) | required | All translations of the name of the rate. |
| `ShortNames` | [Localized text](_objects.md#localized-text) | optional | All translations of the short name of the rate. |
| `ExternalNames` | [Localized text](_objects.md#localized-text) | optional | All translations of the external name of the rate. |
| `Descriptions` | [Localized text](_objects.md#localized-text) | optional | All translations of the description. |
| `PricingType` | [Rate pricing discriminator](rates.md#rate-pricing-discriminator) | required | Rate pricing type. Must match existing pricing type in case of update. |
| `Pricing` | [Rate pricing data parameters](rates.md#rate-pricing-data-parameters) | required | Contains additional data about pricing of the rate. |

### Response

```javascript
{}

> ```

Reviewer 2: @jonas-paul missing response example

Author: Good catch - added!
